test/gc/fiber_ctor.d

```diff
@@ -0,0 +1,14 @@
+// Check avoid of destruction of fiber if GC collection will be invoked during Fiber construction
```

Comment:
`//Check that the Fiber is not also collected when a GC collection is invoked during Fiber construction`